Sources/UIKit/UIColor.swift

```diff
@@ -37,6 +37,27 @@ public extension UIColor {
         self.init(red: components.red, green: components.green, blue: components.blue, alpha: components.alpha)
     }
 
+    var hexString: String {
+
+        var components = UIColor.components(fromHex6: 0)
+        getRed(&components.red, green: &components.green, blue: &components.blue, alpha: &components.alpha)
+        let rgb: Int = (Int)(components.red*255)<<16
+            | (Int)(components.green*255)<<8
+            | (Int)(components.blue*255)<<0
+        return String(format:"#%06x", rgb)
+    }
+
+    var hexStringWithAlpha: String {
+
+        var components = UIColor.components(fromHex6: 0)
+        getRed(&components.red, green: &components.green, blue: &components.blue, alpha: &components.alpha)
+        let argb: Int = (Int)(components.alpha * 255)<<24
```

Comment:
I agree with decide a format and use that format all the way. at the time i did the hex to rgb components, i based my self in rgba, but it seems that this was changed.

Honestly, argb makes more sense to me rather then rbga, so change let's change it 😄 

@mschuetz72 if you prefer, i can open a PR with the change to my bug, if you don't want to worry with that 😉